client/main.lua

```diff
@@ -106,36 +106,28 @@ if Config.UseTarget then
         end
     end)
 else
-    local letSleep = true
-
     CreateThread(function()
```

Comment:
There's 2 CreateThreads when you can bundle them under 1 adds a cleaner code experience